src/app/core/basic-datatypes/configurable-enum/configure-enum-popup/configure-enum-popup.component.spec.ts

```diff
@@ -84,6 +85,48 @@ describe("ConfigureEnumPopupComponent", () => {
     expect(confirmationSpy).toHaveBeenCalledWith(
       "Delete option",
       `Are you sure that you want to delete the option "${male.label}"?`,
+      jasmine.any(Array),
     );
   });
+
+  it("should set newOptionInput with pasted multiline text and prevent default paste", () => {
+    const pastedText = "Option A\nOption B\nOption C";
```

Comment:
maybe add an example of the special logic here also, e.g. includign a duplicate option that we expect to be added only once.